package NDnnProto;


message TNNMFHomeostatisState {
    optional double Pmean = 1;
    optional bool Saturation = 2;
}

message TNNMFHomeostatisConst {
    optional double TargetRate = 1;
    optional double TauMean = 2;
    optional double Alpha = 3;
    optional double Beta = 4;
    optional double Gamma = 5;
    optional double MinWeight = 6;
    optional double MaxWeight = 7;
}

